components/eam/src/dynamics/se/dyn_comp.F90

```diff
@@ -112,6 +112,12 @@ subroutine dyn_init1(fh, NLFileName, dyn_in, dyn_out)
     integer :: npes_se
     integer :: npes_se_stride
 
+#ifdef KOKKOS_TARGET
+#ifdef _OPENMP
+    call endrun( 'kokkos dycore does not run with threads')
```

Comment:
I suggest adding "in this EAM-hommexx configuration" or something like that to the end of this message, since the Kokkos dycore does run with OpenMP threads.

Reply:
thanks, done